sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,78 @@ This product is fully supported and enabled by default as of {product-title} 4.1
+
+* The {sandboxed-containers-operator} pods that use container CPU resource limits to increase the number of available CPUs for the pod might receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resources by using `oc rsh <pod>` and running the `lscpu` command.
++
+[source,terminal]
+----
+$ lscpu
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
+----
++
+The list of available offline CPUs will likely change from run to run in an unpredictable manner.
++
+As a workaround, you can use a pod annotation to request additional CPUs rather than setting a CPU limit. The method of allocating processors is different, and CPUs requested by means of pod annotation are not affected by this issue. Rather than setting a CPU limit, the following `annotation` must be added to the metadata of the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    io.katacontainers.config.hypervisor.default_vcpus: "16"
+----
++
+(link:https://issues.redhat.com/browse/KATA-1376[*KATA-1376*])
+
+* The progress of the runtime installation is shown in the `status` section of the `kataConfig` CR. However, the progress is not shown if all of the following conditions are true:
+
+** The cluster has a machine config pool `worker` without any members (`machinecount=0`).
+** No `kataConfigPoolSelector` is specified to select nodes for installation.
+
++
+In this case, the installation starts on the master nodes because the Operator assumes it is a converged cluster where nodes have both master and worker roles. The `status` section of the `kataConfig` CR is not updated during the installation. (link:https://issues.redhat.com/browse/KATA-1017[*KATA-1017*])
+
+* When using older versions of the buildah tool in {sandboxed-containers-first}, the build fails with the following error:
++
+[source,text]
+----
+process exited with error: fork/exec /bin/sh: no such file or directorysubprocess exited with status 1
+----
++
+You must use the latest version of buildah, available at link:https://quay.io/buildah/stable:latest[quay.io].
++
+(link:https://issues.redhat.com/browse/KATA-1278[*KATA-1278*])
+
+* In the *KataConfig* tab in the web console, if you click *Create KataConfig* while in the *YAML view*, the `KataConfig` YAML is missing the `spec` fields. Toggling to the *Form view* and then back to the *YAML view* fixes this issue and displays the full YAML once again. (link:https://issues.redhat.com/browse/KATA-1372[*KATA-1372*])
+
+* In the *KataConfig* tab in the web console, a `404: Not found` error message appears whether a `KataConfig` custom resource (CR) already exists or not. To access an existing `KataConfig` CR, go to *Home > Search*. From the *Resources* list, select *KataConfig*. (link:https://issues.redhat.com/browse/KATA-1605[*KATA-1605*])
```

Comment:
You could update "Home > Search" so the the ">" is not in bold font. Then, only the UI elements are bold. For example:

> `*Home* > *Search*`